utils/hct/gen_intrin_main.txt

```diff
@@ -706,6 +738,14 @@ $classT [[ro]] SampleBias(in sampler s, in float<3> x, in float bias, in int<3>
 $classT [[]] SampleBias(in sampler s, in float<3> x, in float bias, in int<3> o, in float clamp, out uint_only status) : tex3d_t_bias_o_cl_s;
 $classT [[ro]] SampleGrad(in sampler s, in float<3> x, in $type2 ddx, in $type2 ddy, in int<3> o, in float clamp) : tex3d_t_dd_o_cl;
 $classT [[]] SampleGrad(in sampler s, in float<3> x, in $type2 ddx, in $type2 ddy, in int<3> o, in float clamp, out uint_only status) : tex3d_t_dd_o_cl_s;
+float_like [[]] SampleCmpBias(in sampler_cmp s, in float<3> x, in float compareValue, in float bias) : tex3d_t_comp_bias;
+float_like [[]] SampleCmpBias(in sampler_cmp s, in float<3> x, in float compareValue, in float bias,in int<3> o) : tex3d_t_comp_bias_o;
+float_like [[]] SampleCmpBias(in sampler_cmp s, in float<3> x, in float compareValue, in float bias, in int<3> o, in float clamp) : tex3d_t_comp_bias_o_cl;
+float_like [[]] SampleCmpBias(in sampler_cmp s, in float<3> x, in float compareValue, in float bias, in int<3> o, in float clamp, out uint_only status) : tex3d_t_comp_bias_o_cl_s;
+float_like [[]] SampleCmpGrad(in sampler_cmp s, in float<3> x, in float compareValue, in $type2 ddx, in $type2 ddy) : tex3d_t_comp_dd;
+float_like [[]] SampleCmpGrad(in sampler_cmp s, in float<3> x, in float compareValue, in $type2 ddx, in $type2 ddy,in int<3> o) : tex3d_t_comp_dd_o;
+float_like [[]] SampleCmpGrad(in sampler_cmp s, in float<3> x, in float compareValue, in $type2 ddx, in $type2 ddy,in int<3> o, in float clamp) : tex3d_t_comp_dd_o_cl;
+float_like [[]] SampleCmpGrad(in sampler_cmp s, in float<3> x, in float compareValue, in $type2 ddx, in $type2 ddy,in int<3> o, in float clamp, out uint_only status) : tex3d_t_comp_dd_o_cl_s;
```

Comment:
It's an extremely minor nit, but it's slightly more readible if there's a space after `ddy` and before `in`